contributers/contributer.json

```diff
@@ -83,4 +83,9 @@
         "college" : "IIITA"
     }
 
```

Comment:
Check the format, Its the Array of Objects, So accordingly, format and update PR.